src/utilcode/loaderheap.cpp

```diff
@@ -723,6 +729,9 @@ struct LoaderHeapFreeBlock
             }
 #endif
 
+#if LOADER_HEAP_REDZONE_VALUE != 0
```

Comment:
The zero filling should not be required for `GetGlobalLoaderAllocator()->GetExecutableHeap()`. This method is used in a very few places, and none of them need it.